src/main/java/org/opensearch/flowframework/transport/WorkflowRequest.java

```diff
@@ -150,6 +166,22 @@ public Map<String, String> getParams() {
         return Map.copyOf(this.params);
     }
 
+    /**
+     * Gets the params map
+     * @return the params map
+     */
+    public String getUseCase() {
+        return this.useCase;
+    }
+
+    /**
+     * Gets the params map
+     * @return the params map
+     */
+    public Map<String, String> getDefaultParams() {
+        return Map.copyOf(this.defaultParams);
+    }
```

Comment:
Let's add test for this getters in `WorkflowRequestResponseTests`

Reply:
added